app/models/user.rb

```diff
@@ -10,14 +10,14 @@ def self.all_except(user)
   end
   
   def my_friends
-    User.where(id: self.friendships.true_friends.pluck(:friend_id))
+    User.where(id: friendships.true_friends.pluck(:friend_id))
```

Comment:
why can't we use `friends` relationship?

Reply:
https://stackoverflow.com/questions/20307874/what-is-the-equivalent-of-the-has-many-conditions-option-in-rails-4